lstchain/data/onsite_camera_calibration_param.json

```diff
@@ -31,9 +31,7 @@
      "time_cut_outliers": [2,38]
    },
    "LSTR0Corrections": {
-      "r1_sample_start": 2,
-      "tel_id":1,
-      "r1_sample_end": 38
+      "tel_id":1
```

Comment:
where are the new values:
`"r1_sample_start": 3,`
`"r1_sample_end": 39`
included?

Reply:
Hi @rlopezcoto ,
they are the default values.

Reply:
ok, then I agree not to include them in the input card to avoid confusions in eventual changes in the default ones, thanks!